src/lib/Components/Inbox/Conversations/Messages.tsx

```diff
@@ -26,7 +26,7 @@ interface State {
 const LoadingIndicator = styled.ActivityIndicator`margin-top: 40px;`
 
 export class Messages extends React.Component<Props, State> {
-  constructor(props) {
+  constructor(props: Props) {
```

Comment:
Since we're not using `props` in the constructor could move `state` out and eliminate this entire block

Reply:
Good point! Also a bit above.